apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -419,6 +419,220 @@ public partial class BigtableClient
         }
 
         /// <summary>
+        /// Modifies a row atomically. The method reads the latest existing version
+        /// and value from the specified columns and writes a new entry based on
+        /// pre-defined read/modify/write rules. The new value for the version is the
+        /// greater of the existing version or the current server version. The method
```

Comment:
It's not clear to me what the difference is between "the existing version" and "the current server version". Can we clarify that?

Reply:
Done. Let me know if that is still unclear.